### PR TITLE
Request city form options

### PR DESCRIPTION
**Pull request by gabbygermanson**
Request form locations, filename changes, documentation on all files, basic styling changes.

Basically all files edited, most either name change or minor styling / documentation.

### CODE REVIEW

File: src/main/java/com/google/step/BackendQueryDatastore.java [hunk 1]
```diff
@@ -23,25 +23,26 @@
 import java.util.stream.Collectors;
 
 /***
-    This servlet retrieves the mapImage metadata (location, zoom level, etc.) from Datastore.
+    This servlet retrieves the tracked metadata (location and coordinates) from Datastore.
+    This metadata along with a range of zoom levels is used to initiate new MapImage instances representing new map snapshots.
+    The MapImage instances are sent
```
Comment (cgregori) [inline src/main/java/com/google/step/BackendQueryDatastore.java:28]: MapImage instances are sent to SaveImageCloud.java?

File: src/main/java/com/google/step/SaveMapImageDatastore.java [hunk 2]
```diff
@@ -1,58 +1,36 @@
...
-import com.google.api.services.drive.model.FileList;
 import java.util.Arrays;
 import java.util.ArrayList;
-import java.util.List;
 import com.google.gson.Gson;
 import com.google.gson.reflect.TypeToken;
 
-// TODO: Tests
+/***
+    This servlet retrieves MapImages after many attributes are set in SaveImageCloud.java to be stored in Datastore.
```
Comment (cgregori) [inline src/main/java/com/google/step/SaveMapImageDatastore.java:19]: Is SaveImageCloud.java in this PR? I don't see it in the changed files or in my local repo right now

File: src/main/webapp/script.js [hunk 3]
```diff
@@ -1,20 +1,41 @@
...
+            for (var i = 0; i < array.length; i++) {
                 const url = array[i].url;
                 // TODO: create entire image list structure
                 $("#requestedImages").append(`<li><img src="${url}"></li>`);
             }
         }
     });
-    
-});
+
```
Comment (cgregori) [inline src/main/webapp/script.js:13]: Maybe add a comment here about how this fetch loads the Locations options for the form through Datastore

File: src/main/java/com/google/step/SaveMapImageDatastore.java (hunk 2 above)
Comment (gabbygermanson) [reply src/main/java/com/google/step/SaveMapImageDatastore.java:19]: Revived it, thank you!

File: src/main/webapp/script.js (hunk 3 above)
Comment (gabbygermanson) [reply src/main/webapp/script.js:13]: Good idea

File: src/main/java/com/google/step/BackendQueryDatastore.java [hunk 4]
```diff
@@ -22,26 +22,27 @@
 import java.io.InputStreamReader;
 import java.util.stream.Collectors;
 
-/***
-    This servlet retrieves the mapImage metadata (location, zoom level, etc.) from Datastore.
-    A GET request gets TrackedLocations so request URLs can be constructed and sent to Static Maps.
+/***MapImage instances are sent to SaveImageCloud.java?
```
Comment (theannielin) [inline src/main/java/com/google/step/BackendQueryDatastore.java:25]: Add space here or new line

**Pull request merged**